server/autotest_server/testers/ai/requirements.txt

```diff
@@ -0,0 +1,2 @@
+python-dotenv
+git+https://github.com/MarkUsProject/ai-autograding-feedback.git@main#egg=ai_feedback
```

Comment:
I don't think the `egg` part is necessary. But also, it's not very robust to always install the latest version from GitHub.

Please alter this approach: instead of including `ai_feedback` in the `requirements.txt`, include a configuration option `ai_feedback_version` (at/near the top of the test settings) where the user can enter a string. Then in the test environment setup (look at the Python tester's `setup.py` `create_environment` function) do a pip install of the ai feedback repo, but put in whatever the user specifies after the `@`. This will allow the user to specify a certain branch, tag, or commit hash for installation.